src/transformers/pipelines/image_feature_extraction.py

```diff
@@ -59,14 +66,23 @@ def _forward(self, model_inputs):
         model_outputs = self.model(**model_inputs)
         return model_outputs
 
-    def postprocess(self, model_outputs, return_tensors=False):
-        # [0] is the first available tensor, logits or last_hidden_state.
+    def postprocess(self, model_outputs, pool=None, return_tensors=False):
+        pool = pool if pool is not None else False
+
+        if pool:
+            if "pooler_output" not in model_outputs:
+                raise ValueError("The model needs to have a `pooler` layer to use the `pool` option.")
```

Comment:
it more like the pooler_outpus were not returned since we don't check if there is a pooler layer layer or not 😉